src/sentry/eventstream/snuba.py

```diff
@@ -147,9 +147,11 @@ def start_delete_groups(self, project_id, group_ids):
 
         return state
 
-    def end_delete_groups(self, state):
+    def end_delete_groups(self, state, cutoff_datetime=None):
```

Comment:
Is this parameter still being used? It seems that this is dead code and can be reverted.

Reply:
damn, yeah will fix